src/packagedcode/models.py

```diff
@@ -711,8 +711,34 @@ class PackageData(IdentifiablePackageData):
         repr=True,
     )
 
+    holder= String(
+    label='Holder',
+    help='Holder for this package. Typically one per line.')
+
     def __attrs_post_init__(self, *args, **kwargs):
         self.populate_license_fields()
+        self.populate_holder_field()
+
+    def populate_holder_field(self):
+        if self.copyright:
+            from cluecode.copyrights import CopyrightDetector
+
+            numbered_lines = [
+                (count, value) for count, value in enumerate(self.copyright.split("\n"), start=1)
+            ]
+            detector = CopyrightDetector()
+            holder_list = list(
+                detector.detect(
+                    numbered_lines,
+                    include_copyrights=False,
+                    include_holders=True,
+                    include_authors=False,
+                )
+            )
+            if holder_list:
+                self.holder = "\n".join(
```

Comment:
We are joining copyright holders with newlines, would it make more sense to have this as a list? 
Also note that copyrights also is a single string, and if we make holders a list, we should have the same there too.

This would be moving towards better package data like we did for licenses. Moving from just a expression string to objects with details. (Though we don't need any more details for copyrights holders, as start/end line don't apply here like files.)

Reply:
I chatted with @pombredanne, and he suggested that we stick to the current string format because shifting to a list will create a whole lot of other issues, such as how to serialize those lists..